test/sol6/reserves/kyberFprReserveV2.js

```diff
@@ -0,0 +1,3422 @@
+const MockConversionRates = artifacts.require("MockConversionRates");
+const TestToken = artifacts.require("TestToken");
+const WethToken = artifacts.require("WethToken");
+const Reserve = artifacts.require("KyberFprReserveV2");
+const MockSanityRates = artifacts.require("MockSanityRates");
+const NoPayableFallback = artifacts.require("NoPayableFallback");
+const SanityRatesGasPrice = artifacts.require("SanityRatesGasPrice");
+
+const Helper = require("../../helper.js");
+const reserveSetup = require("../../reserveSetup.js");
+const BN = web3.utils.BN;
+
+//global variables
+//////////////////
+const {precisionUnits, ethDecimals, ethAddress, zeroAddress, zeroBN, MAX_ALLOWANCE} = require("../../helper.js");
+const {expectEvent, expectRevert} = require('@openzeppelin/test-helpers');
+
+//balances
+let expectedReserveBalanceWei = new BN(0);
+let expectedReserveBalanceWeth = new BN(0);
+let reserveTokenBalance = [];
+let reserveTokenImbalance = [];
+let minimalRecordResolution;
+
+//permission groups
+let admin;
+let operator;
+let alerter;
+let network;
+let withdrawAddress;
+
+//contracts
+let weth;
+let convRatesInst;
+let reserveInst;
+
+//block data
+let currentBlock;
+let maxGasPrice = new BN(150).mul(new BN(10).pow(new BN(9))); // 150 * 10^9
+let doRateValidation = true;
+
+//tokens data
+////////////
+let numTokens = 5;
+let tokens = [];
+let tokenDecimals = [];
+let tokenAdd = [];
+
+//base buy and sell rates (prices)
+let baseBuyRate = [];
+let baseSellRate = [];
+
+//imbalance buy and sell steps, check values in reserveSetup file
+let imbalanceBuyStepX = [];
+let imbalanceBuyStepY = [];
+let imbalanceSellStepX = [];
+let imbalanceSellStepY = [];
+
+//compact datam  check values in reserveSetup file
+let compactBuyArr = [];
+let compactSellArr = [];
+
+// v1 data
+let qtyBuyStepX = [];
+let qtyBuyStepY = [];
+let qtySellStepX = [];
+let qtySellStepY = [];
+
+contract('KyberFprReserveV2', function(accounts) {
+	before("Global setup", async function () {
+		// set account addresses
+		admin = accounts[0];
+		operator = accounts[1];
+		network = accounts[2];
+		user1 = accounts[4];
+		withdrawAddress = accounts[6];
+		alerter = accounts[8];
+		walletForToken = accounts[9];
+
+		currentBlock = await Helper.getCurrentBlock();
+		weth = await WethToken.new("WrapETH", "WETH", 18);
+
+		// create tokens
+		for (let i = 0; i < numTokens; ++i) {
+			tokenDecimals[i] = new BN(18);
```

Comment:
somehow here its linted with 4 spaces tab.
and later with 2.

can run prettier on full file.